server/models/video/video-abuse.ts

```diff
@@ -7,14 +9,164 @@ import {
+              { deletedVideo: { channel: { displayName: { [Op.iLike]: `%${options.search}%` } } } }
+            ]
+          },
+          { '$Account.name$': { [Op.iLike]: `%${options.search}%` } }
+        ]
+      })
+    }
 
+    return {
+      attributes: {
+        include: [
+          [
+            literal(
+              '(' +
+                'SELECT t.count ' +
+                'FROM ( ' +
+                  'SELECT id, ' +
+                         'count(id) OVER (PARTITION BY "videoId") ' +
+                  'FROM "videoAbuse" ' +
+                ') t ' +
+                'WHERE t.id = "VideoAbuseModel".id ' +
+              ')'
+            ),
+            'countReportsForVideo'
+          ],
+          [
+            literal(
+              '(' +
+                'SELECT t.nth ' +
+                'FROM ( ' +
+                  'SELECT id, ' +
+                         'row_number() OVER (PARTITION BY "videoId" ORDER BY "createdAt") AS nth ' +
+                  'FROM "videoAbuse" ' +
+                ') t ' +
+                'WHERE t.id = "VideoAbuseModel".id ' +
+              ')'
+            ),
+            'nthReportForVideo'
+          ],
+          [
+            literal(
+              '(' +
+                'SELECT count("videoAbuse"."id") ' +
+                'FROM "videoAbuse" ' +
+                'INNER JOIN "video" ON "video"."id" = "videoAbuse"."videoId" ' +
+                'INNER JOIN "videoChannel" ON "videoChannel"."id" = "video"."channelId" ' +
+                'INNER JOIN "account" ON "videoChannel"."accountId" = "account"."id" ' +
+                'WHERE "account"."id" = "VideoAbuseModel"."reporterAccountId" ' +
+              ')'
+            ),
+            'countReportsForReporter'
+          ],
+          [
+            literal(
+              '(' +
+                'WITH ' +
+                  'ids AS ( ' +
+                    'SELECT "account"."id" ' +
+                    'FROM "account" ' +
+                    'INNER JOIN "videoChannel" ON "videoChannel"."accountId" = "account"."id" ' +
+                    'INNER JOIN "video" ON "video"."channelId" = "videoChannel"."id" ' +
+                    'WHERE "video"."id" = "VideoAbuseModel"."videoId" ' +
+                  ') ' +
+                'SELECT count("videoAbuse"."id") ' +
+                'FROM "videoAbuse" ' +
+                'INNER JOIN "video" ON "video"."id" = "videoAbuse"."videoId" ' +
+                'INNER JOIN "videoChannel" ON "videoChannel"."id" = "video"."channelId" ' +
+                'INNER JOIN "account" ON "videoChannel"."accountId" = "account"."id" ' +
+                'INNER JOIN ids ON "account"."id" = ids.id ' +
+              ')'
+            ),
+            'countReportsForReportee'
+          ]
+        ]
+      },
+      include: [
+        {
+          model: AccountModel,
+          required: true,
+          where: { ...searchAttribute(options.searchReporter, 'name') }
```

Comment:
`where: searchAttribute(options.searchReporter, 'name')`

Reply:
ah. of course :octopus: